google/resource_compute_instance_group_manager.go

```diff
@@ -200,6 +217,7 @@ func flattenNamedPorts(namedPorts []*compute.NamedPort) []map[string]interface{}
 }
 
 func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interface{}) error {
+	computeApiVersion := v1
```

Comment:
Might be worth it to add a comment here that this will eventually be figured out based on what's set in the request with a link to the issue about multiple api versions